SpatialGDK/Documentation/content/porting-unreal-project-to-gdk.md

```diff
@@ -113,63 +128,68 @@ etc...
-          },
-          {
-            "global_component_streaming_query": {
-              "component_name": "improbable.unreal.generated.yourprojectgamestate.YourProjectGameStateMultiClientRepData"
-            }
-          },
-          {
-            "global_component_streaming_query": {
-              "component_name": "improbable.unreal.generated.yourprojectgamemode.YourProjectGameModeMultiClientRepData"
+            "streaming_query": [
+            {
+                "global_component_streaming_query": {
+                "component_name": "improbable.unreal.GlobalStateManager"
+                }
+            },
+            {
+                "global_component_streaming_query": {
+                "component_name": "improbable.unreal.generated.yourprojectgamestate.YourProjectGameStateMultiClientRepData"
+                }
+            },
+            {
+                "global_component_streaming_query": {
+                "component_name": "improbable.unreal.generated.yourprojectgamemode.YourProjectGameModeMultiClientRepData"
+                }
             }
-          }
-        ],
-        
+            ],
     
 ## Generating code for SpatialOS C++ workers and building your project
-1. Set your Unreal project to work with the SpatialOS Unreal GDK fork of the Unreal Engine, which you cloned and installed earlier. To do this:
-    1. In File Explorer, navigate to `<ProjectRoot>\<GameRoot>`.
-    1. Right-click your uproject file and select **Switch Unreal Engine version**.
-    1. Select the path to the Unreal Engine fork you cloned earlier.
-    1. In your terminal, from `Plugins\<UnrealGDKRoot>\SpatialGDK\Build\Scripts`, run `BuildWorker.bat <YourProject>Editor Win64 Development <YourProject>.uproject`
-    1. In File Explorer, right-click your uproject file and select **Generate Visual Studio project files**.
-    1. Open the solution in Visual Studio, and compile and run the project in the Development Editor configuration.
-
-## Running the Schema Generator and setting up the UE4 Editor
-1. In the Unreal Editor, on the SpatialOS Unreal GDK toolbar, click **Generate Schema** to run the `Schema Generator`.
-1. On the same toolbar, click **Snapshot**.
+Set your Unreal project to work with the SpatialOS GDK for Unreal fork of the Unreal Engine, which you cloned and installed earlier. To do this:
+1. In File Explorer, navigate to `<ProjectRoot>\<GameRoot>`.
+1. Right-click your `.uproject` file and select **Switch Unreal Engine version**.
+1. Select the path to the Unreal Engine fork you cloned earlier.
+1. In a terminal window, from `<ProjectRoot>\<GameRoot>\Plugins\UnrealGDK\SpatialGDK\Build\Scripts`, run `BuildWorker.bat <YourProject>Editor Win64 Development <YourProject>.uproject`   
+For Example "`BuildWorker.bat StarterProjectEditor Win64 Development StarterProject.uproject`"  
+This allows you to run locally managed workers. For example running a local spatial deployment with `one_worker` or `two_worker` launch configurations.
```

Comment:
We are introducing "locally managed workers" which doesn't mean anything to users. Have suggested adding something to the beginning of this section.
Original:
"This allows you to run locally managed workers. For example running a local spatial deployment with `one_worker` or `two_worker` launch configurations."
->
->This doesn't mean much to me as I port my project - why would I want to do this? I assume this means using two local server-workers. Also a "launch configuration" doesn't mean much to me. Also we say "server-workers" rather than managed workers.
-> 
Revised (taking into account the additional text at the top of this section):
This means you can now run a local deployment with more than one server; your [launch configuration]({{URLRoot}}/content/glossary#launch-configuration) can have two servers-workers.